<launch>
 <!-- Node Settings -->
    <arg name="lepton_on" default="false"/>
    <arg name="tamarisk_on" default="false"/>
    <arg name="bluefox_right_on" default="true"/>
    <arg name="bluefox_left_on" default="false"/>

<!-- Save on Command Node bluefox right-->
<node if="$(arg bluefox_right_on)"
name="bluefox_right_image_server" pkg="image_sender" type="img_sender_server" output="screen">
	<param name="identifier" type="string" value="/mv_25000060/image_raw"/>
</node>
</launch>